--- Makefile.am.bak	2014-11-20 14:03:46.000000000 -0600
+++ Makefile.am	2017-10-26 00:13:50.342442463 -0500
@@ -13,9 +13,9 @@
 CXXFLAGS_DEBUG = -g3 -O0  -Wall -W -Wcast-align -Werror
 TEST_COV_FLAGS = -ftest-coverage -fprofile-arcs
 
-noinst_LTLIBRARIES = libfreeform.la
+lib_LTLIBRARIES = libfreeform.la
 
-noinst_PROGRAMS = readfile newform chkform bufform checkvar
+bin_PROGRAMS = readfile newform chkform bufform checkvar
 
 libfreeform_la_SOURCES = $(FF_SRCS) $(FF_HDRS)
 
